trunk/SUAVE/Components/Energy/Networks/Lift_Cruise.py

```diff
@@ -252,41 +261,76 @@ def evaluate_thrust(self,state):
         battery.energy_calc(numerics)   
         
         # Pack the conditions
+        a                    = conditions.freestream.speed_of_sound
+        R_lift               = propeller_lift.tip_radius
+        R_forward            = propeller_forward.tip_radius
         rpm_lift             = motor_lift.outputs.omega*60./(2.*np.pi)
```

Comment:
There are a bunch of hard coded unit conversions in this function. Please replace with standard units method.

Reply:
Standard Units used